packages/core/src/auditor.ts

```diff
@@ -0,0 +1,42 @@
+import { Storable } from "./storage";
+
+export type AuditKind = "auth" | "account" | "org" | "vault" | "item";
+export type AuditChangeType = "created" | "updated" | "deleted";
```

Comment:
Anything critical you think I missed for this initial version, @MaKleSoft ?

Reply:
Instead of restricting ourselves to specific CRUD actions, I think we should go for something more generic, analogous to what we have in [the logging module](https://github.com/padloc/padloc/blob/main/packages/core/src/logging.ts#L16-L24). In fact, now that I think of it, the logging module already pretty much does what we're trying to do, no? All we need is an endpoint to send events from the client (for those events that we can't catch on the server, like editing/viewing items, plus some logic around storing encrypted data in events.

Reply:
You can see in my latest commit that I tried the endpoint approach and it was becoming a problem.

Reply:
Ok, so maybe we should first define what those need are. I think we should have a brief call about this.